### ISSUE TITLE
Access-Control-Allow-Origin header is sent twice

### ISSUE THREAD

**Issue by PedroATavares** [CLARIAH/grlc#265]
Calls using the `api-git` location responds with multiple `Access-Control-Allow-Origin` headers making client side request unusable.
An example call:
```
curl "http://grlc.io/api-git/albertmeronyo/my-linked-data-api/myquery" -H "accept: text/csv" -v
*   Trying 213.187.244.59...
* TCP_NODELAY set
* Connected to grlc.io (213.187.244.59) port 80 (#0)
> GET /api-git/albertmeronyo/my-linked-data-api/myquery HTTP/1.1
> Host: grlc.io
> User-Agent: curl/7.58.0
> accept: text/csv
> 
< HTTP/1.1 200 OK
< Date: Thu, 14 May 2020 15:19:13 GMT
< Content-Type: text/csv; charset=UTF-8
< Content-Length: 2355
< Connection: keep-alive
< Server: gunicorn/19.6.0
< Access-Control-Allow-Origin: *
< Access-Control-Allow-Origin: *
< X-Proxy-Cache: MISS
< 
"band_label"
"42-01"
"9xdead"
"A Thousand Horses"
"Alianza (Argentine band)"
"Alkbottle"

...
* Connection #0 to host grlc.io left intact
```

As seen in the example the header `Access-Control-Allow-Origin: *` is sent twice.
This might be due to the nginx configuration on https://github.com/CLARIAH/grlc/blob/448621cc5dfe7e8a3fc26e5419049481427d8222/docker-assets/assets/runtime/configs/nginx/grlc#L55

**Comment by c-martinez** [comment]
Hi @PedroATavares -- well spotted. We recently added the header also in the python code, which makes the header in nginx redundant. I will fix this.

I'm just curious, why does this make the request on the client side unusable?

**Comment by PedroATavares** [comment]
Hi @c-martinez, thank you for taking a look at this so quickly.

To answer your question, if the calls are coming from the client running in the browser and the response contains multiple `Access-Control-Allow-Origin` headers the browser will discard the response with the following error:
```
Access to XMLHttpRequest at 'http://grlc.io/api-git/albertmeronyo/my-linked-data-api/myquery' from origin 'http://localhost:8080' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header contains multiple values 'http://localhost:8080, *', but only one is allowed.
```

**Comment by c-martinez** [comment]
You are right @PedroATavares, I've deployed this change on dev.grlc.io, if you could test that everything works as expected there, then I will patch grlc.io as well.

Thanks!

**Comment by PedroATavares** [comment]
@c-martinez I checked and everything works as expected in dev.grlc.io, please let me know once its been patched in grlc.io. Thank you for the quick fix.

**Comment by eknutov** [comment]
@c-martinez thanks for the quick fix, we are using `grlc` a lot for some projects and it is nice to have everything fixed. thanks.

**Comment by c-martinez** [comment]
@PedroATavares -- grlc.io has been patched. I will close this issue but please reopen it if the problem persists.

@eknutov -- nice to hear you are using grlc for some projects. If you could share with us what projects you are using grlc for, it is always interesting to hear from users. Even better, if you could help us by citing grlc on your work, that would be fantastic!

**Comment by eknutov** [comment]
@c-martinez it is a Genealogy project we are using it for at the moment (https://genealogie.semaku.com) It visualizes timelines and relationships using some Dutch Achieve Open Data. We're definitely citing projects we use, so no worries.

**Comment by c-martinez** [comment]
Really interesting! Thanks for sharing @eknutov !
